reference-architecture/azure-ansible/bastion.sh

```diff
@@ -294,48 +207,18 @@ cat > /home/${AUSERNAME}/setup-azure-node.yml <<EOF
           "tenantID" : "{{ g_tenantId }}",
           "resourceGroup": "{{ g_resourceGroup }}",
         }
-    notify:
-    - restart atomic-openshift-node
-  - name: insert the azure disk config into the node
-    modify_yaml:
-      dest: "{{ node_conf }}"
-      yaml_key: "{{ item.key }}"
-      yaml_value: "{{ item.value }}"
-    with_items:
-    - key: kubeletArguments.cloud-config
-      value:
-      - "{{ azure_conf }}"
-
-    - key: kubeletArguments.cloud-provider
-      value:
-      - azure
-    notify:
-    - restart atomic-openshift-node
-  - name: delete the node so it can recreate itself
-    command: oc delete node {{inventory_hostname}}
-    delegate_to: bastion
-  - name: sleep to let node come back to life
-    pause:
-       minutes: 1
-- hosts: masters
-  tasks:
-  - name: sleep to make sure everything up
-    pause:
-       minutes: 3
-  - name: make sure its set to unsceduable
-    command: oadm manage-node {{inventory_hostname}} --schedulable=false
-    delegate_to: master1
 EOF
 
-
-
 cat <<EOF > /etc/ansible/hosts
 [OSEv3:children]
 masters
 nodes
 etcd
 
 [OSEv3:vars]
+osm_controller_args={'cloud-provider': ['azure'], 'cloud-config': ['/etc/azure/azure.conf']}
```

Comment:
Just in case, according to this[1], containerized installations mount /etc/origin and /var/lib/origin only, so they recommend to store cloud files in those folders.

[1] https://docs.openshift.com/container-platform/3.5/install_config/configuring_openstack.html#openstack-configuring-nodes-manually